test/tests/api-cit/v2_0/lookups_tests.py

```diff
@@ -28,14 +28,14 @@ def setUpClass(cls):
             "macAddress": "ae:aa:aa:aa:aa:aa",
             "node": "456"
         }
-        cls.id = ""
+        cls.lookup_id = ""
         cls.patchedNode = {"node": "666"}
 
     def test_check_lookups(self):
         # """ Testing GET:/lookups """
         Api().lookups_get()
         rsp = self.__client.last_response
-        logs.debug("Lookup list: %s", dumps(rsp.data, indent=4))
+        # logs.debug("Lookup list: %s", dumps(rsp.data, indent=4))
```

Comment:
added back in at debug_7 level, output can be extensive, so only when you really need it.